packages/e2e-test-utils/src/site-editor.js

```diff
@@ -10,7 +10,7 @@ import { addQueryArgs } from '@wordpress/url';
 
 const SELECTORS = {
 	visualEditor: '.edit-site-visual-editor iframe',
-	loadingSpinner: '.edit-site-canvas-spinner',
+	loadingProgressBar: '.edit-site-canvas-loader',
```

Comment:
Nit: rename to `canvasLoader`.

Reply:
Done in 6b309f0c6b0909e257ee14a159d6eaf8920f7070